### ISSUE TITLE
ANR is occurring when releasing exoplayer only on V30 device

### ISSUE THREAD

**Issue by hixguru** [google/ExoPlayer#4341]
### Issue description
Recently, ANR is occurring when releasing exoplayer only on V30 device.

### Reproduction steps
sorry. can not reproduce because there is no device.

### Link to test content
Provide a link to media that reproduces the issue. If you don't wish to post it
publicly, please submit the issue, then email the link to
dev.exoplayer@gmail.com using a subject in the format "Issue #1234".

### Version of ExoPlayer being used
- v2.6.0

### Device(s) and version(s) of Android being used
- device : LG-V30
- version: 8.0.0

### A full bug report captured from the device
we received these error reports like below
- **Input dispatching timed out (Waiting because the touched window's input channel is full. Outbound queue length: 1. Wait queue length: 52.)** 
- **executing service co.ri...**
```
"main" tid=1 Waiting 
"main" prio=5 tid=1 Waiting
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x72f25df0 self=0x7a854c7a00
  | sysTid=22065 nice=-10 cgrp=default sched=0/0 handle=0x7a8a1129b0
  | state=S schedstat=( 0 0 0 ) utm=1053 stm=109 core=6 HZ=100
  | stack=0x7fddf4a000-0x7fddf4c000 stackSize=8MB
  | held mutexes=
  at java.lang.Object.wait (Object.java)
- waiting on <0x07dc1094> (a com.google.android.exoplayer2.k)
  at com.google.android.exoplayer2.ExoPlayerImplInternal.release (ExoPlayerImplInternal.java:240)
- locked <0x07dc1094> (a com.google.android.exoplayer2.k)
  at com.google.android.exoplayer2.ExoPlayerImpl.release (ExoPlayerImpl.java:300)
  at com.google.android.exoplayer2.SimpleExoPlayer.release (SimpleExoPlayer.java:697)
  at co.riiid.vida.player.Player.release (Player.java:102)
  at co.riiid.vida.paper.BasePaperFragment.onDestroy (BasePaperFragment.java:128)
  at android.support.v4.app.Fragment.performDestroy (Fragment.java:2585)
  at android.support.v4.app.FragmentManagerImpl.moveToState (FragmentManagerImpl.java:1559)
  at android.support.v4.app.FragmentManagerImpl$2$1.run (FragmentManagerImpl.java:1620)
  at android.os.Handler.handleCallback (Handler.java:789)
  at android.os.Handler.dispatchMessage (Handler.java:98)
  at android.os.Looper.loop (Looper.java:164)
  at android.app.ActivityThread.main (ActivityThread.java:6709)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.android.internal.os.Zygote$MethodAndArgsCaller.run (Zygote.java:240)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:769)
```

I've tried a lot of debugging to solve this problem, but I'm having a hard time

**Comment by gb103** [comment]
@ojw28 
Do we have any updates about this issue, as this is not only one device specific, it is producible on many devices in my case. Same error logs can be seen as above mentioned.

**Comment by ojw28** [comment]
Marking as a duplicate of https://github.com/google/ExoPlayer/issues/4352.
